pandas_log/extras.py

```diff
@@ -0,0 +1,165 @@
+import pandas as pd
+import pandas_flavor as pf
+
+
+@pf.register_dataframe_method
+def dq_check(df, percentiles=None, thorough=True):
```

Comment:
@cmdavis4 I dont think its relate to the repo purpose, I think you can use pandas-profiling to get all these stats and more (use the private method describe pandas_profiling.model.describe import describe )

Reply:
I didn't know that package existed, I have been looking for something like that! I thought this fit in terms of how I've been using this package, but you're right it's not strictly additional logging for existing pandas methods. I'll remove it.